client/src/test/java/org/apache/rocketmq/client/ClientConfigTest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.rocketmq.client;
+
+import org.apache.rocketmq.common.message.MessageQueue;
+import org.apache.rocketmq.remoting.protocol.LanguageCode;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClientConfigTest {
+
+    private ClientConfig clientConfig;
+
+    private final String resource = "resource";
+
+    @Before
+    public void init() {
+        clientConfig = createClientConfig();
+    }
+
+    @Test
+    public void assertClientConfig() {
+        assertEquals("unitName", clientConfig.getUnitName());
+        assertEquals("127.0.0.1", clientConfig.getClientIP());
+        assertEquals(1, clientConfig.getClientCallbackExecutorThreads());
+        assertEquals(1000 * 30, clientConfig.getPollNameServerInterval());
+        assertEquals(1000 * 30, clientConfig.getHeartbeatBrokerInterval());
+        assertEquals(1000 * 5, clientConfig.getPersistConsumerOffsetInterval());
+        assertEquals(1000, clientConfig.getPullTimeDelayMillsWhenException());
+        assertEquals("{}", clientConfig.getSocksProxyConfig());
+        assertEquals(LanguageCode.JAVA, clientConfig.getLanguage());
+        assertEquals(AccessChannel.LOCAL, clientConfig.getAccessChannel());
+        assertEquals(1000 * 3, clientConfig.getMqClientApiTimeout());
+        assertEquals(200, clientConfig.getDetectTimeout());
+        assertEquals(1000 * 2, clientConfig.getDetectInterval());
+        assertTrue(clientConfig.isUnitMode());
+        assertTrue(clientConfig.isDecodeReadBody());
+        assertTrue(clientConfig.isDecodeDecompressBody());
+        assertTrue(clientConfig.isEnableStreamRequestType());
+        assertTrue(clientConfig.isSendLatencyEnable());
+        assertTrue(clientConfig.isEnableHeartbeatChannelEventListener());
+        assertFalse(clientConfig.isUseHeartbeatV2());
+    }
```

Comment:
This test doesn't seem to make much sense

Reply:
> This test doesn't seem to make much sense

OK, I'll remove it.